docs/README.md

```diff
@@ -25,6 +25,7 @@ The main features of MSTest v2 are documented through a list of RFCs:
 - [DynamicData Attribute for Data Driven Tests](RFCs/006-DynamicData-Attribute.md)
 - [DataSource Attribute Vs ITestDataSource](RFCs/007-DataSource-Attribute-VS-ITestDataSource.md)
 - [Test case timeout via runsettings](RFCs/008-TestCase-Timeout.md)
+- [MSTest element via runsettings](https://learn.microsoft.com/en-us/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file?view=vs-2019#mstest-element)
```

Comment:
I'd recommend you to always use the link without the culture + whenever possible (i.e. the page is not linked to a specific TFM or VS) omit TFM or VS specific version.

```suggestion
- [MSTest element via runsettings](https://learn.microsoft.com/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file#mstest-element)
```